xapian-applications/omega/omindextest

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+# omindextest: Test omindex
+#
+# Copyright (C) 2019 Bruno Baruffaldi
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+# USA
+
+set -e
+
+srcdir=`echo "$0"|sed 's!/*[^/]*$!!'`
+TEST_FILES="$srcdir/testfiles/"
+TEST_DB="testdatabase"
+
+trap 'rm -rf "$TEST_DB"' 0 1 2 13 15
+
+./omindex --db "$TEST_DB" --url="/$TEST_FILES" "$TEST_FILES"
```

Comment:
I think this should use `--overwrite` too for robustness - we really want to always start from a new empty database or else the user may get unexpected problems if there's an old `testdatabase` left behind from a previous run.

Reply:
I agree, I am going to add it.